packages/api-specification/generate-docs.js

```diff
@@ -70,6 +70,12 @@ const generateInterface = (interfaceObj) => {
 const generateProperty = (prop, className) => {
   const typeArgsString = getType(prop.type);
   mdObject[className] += `**${prop.name}${prop.flags.isOptional ? '?' : ''}**: \`${typeArgsString}\`  \n`;
+  const comment = prop.comment;
+  if (comment) {
+    if (comment.shortText) {
```

Comment:
Could we do `if (comment && comment.shortText)`, or are there other fields we might want in the future?